### PR TITLE
redirect stuff

### PR DESCRIPTION
**Pull request by devsnek**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
+  <meta charset=utf-8 />
+  <meta name=viewport content="width=device-width, initial-scale=1" />
+  <link rel="shortcut icon" type=image/x-icon href=/static/favicon.ico />
+  <link rel=stylesheet type=text/css href=//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.8.0/styles/default.min.css />
+  <link rel=stylesheet type=text/css href=//cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css />
+  <script type=text/javascript src=//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.8.0/highlight.min.js async></script>
 </head>
 <body>
-  <div id="app"></div>
+  <div id=app></div>
```
Comment (hydrabolt) [inline index.html]: No put the quotation marks back to all of this

File: build/webpack.dev.conf.js [hunk 2]
```diff
@@ -15,8 +15,7 @@ module.exports = merge(baseWebpackConfig, {
   module: {
     rules: utils.styleLoaders({ sourceMap: config.dev.cssSourceMap })
   },
-  // cheap-module-eval-source-map is faster for development
-  devtool: '#cheap-module-eval-source-map',
```
Comment (GusCaplan) [inline build/webpack.dev.conf.js]: this broke source maps so i changed it to the other one

File: package.json [hunk 3]
```diff
@@ -4,7 +4,8 @@
   "description": "The website housing documentation for discord.js",
   "author": "Amish Shah <amishshah.2k@gmail.com>",
   "contributors": [
-    "Schuyler Cebulskie <Gawdl3y@gawdl3y.com>"
+    "Schuyler Cebulskie <Gawdl3y@gawdl3y.com>",
+    "snek <me@gus.host>"
```
Comment (hydrabolt) [inline package.json]: If you're going to add yourself to contrib at least capitalise snek :(

Comment (GusCaplan) [inline package.json]: it is lowercase on purpose

File: index.html (hunk 1 above)
Comment (GusCaplan) [inline index.html]: plzno

**Comment by devsnek** [comment]
@hydrabolt @Gawdl3y

File: index.html [hunk 4]
```diff
@@ -10,6 +10,6 @@
   <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js" async></script>
 </head>
 <body>
-  <div id="app"></div>
+  <div id=app></div>
```
Comment (hydrabolt) [inline index.html]: no

File: src/components/Stats.vue [hunk 5]
```diff
@@ -1,42 +1,42 @@
...
+        if (!err) data.stars = res.body.stargazers_count.toLocaleString();
+      });
+      request.get('https://api.github.com/repos/hydrabolt/discord.js/stats/contributors').end((err, res) => {
+        if (!err) data.contributors = res.body.length.toLocaleString();
+      });
+
+      return data;
+    },
+  };
+</script>
```
Comment (hydrabolt) [inline src/components/Stats.vue]: fix pls

File: src/components/UnknownPage.vue [hunk 6]
```diff
@@ -1,7 +1,7 @@
 <template>
   <div>
     <h1>Unknown {{ type || 'page' }}</h1>
-    <p>This page doesn't actually exist. Oh no!</p>
+    <p>Oh no! This page doesn't actually exist.</p>
```
Comment (hydrabolt) [inline src/components/UnknownPage.vue]: It sounded more sarcastic before this change :\

**Pull request closed without merging**